### PR TITLE
Riak_test driver of the CS client tests

### PR DESCRIPTION
**Pull request by slfritchie**
This is the minimal (?) amount necessary to get the client tests (Clojure, Python, Erlang) tests wrapped up and have `riak_test` drive them.

### CODE REVIEW

**Comment by slfritchie** [comment]
Reviewer: please add your documentation wish list for Santa Claus.

File: riak_test/src/rtcs.erl [hunk 1]
```diff
@@ -25,7 +25,10 @@
 -define(DEFAULT_PROTO, "http").
 
 setup(NumNodes) ->
-    Nodes = build_cluster(NumNodes),
+    setup(NumNodes, ee_config(), stanchion_config(), cs_config()).
+
+setup(NumNodes, EE_config, Stanchion_config, CS_config) ->
```
Comment (kellymclaughlin) [inline riak_test/src/rtcs.erl]: `EE_config`, `CS_config`, etc make it difficult to distinguish from the `ee_config()` and `cs_config()` function calls when reading. Can we change that to be camel-cased like `EEConfig`?

File: riak_test/tests/external_client_tests.erl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    FirstNode = hd(RiakNodes),
+
+    {AccessKeyId, SecretAccessKey} = rtcs:create_user(FirstNode, 1),
+
+    %% User config
+    CS_http_port = rtcs:cs_port(FirstNode),
+    _UserConfig = rtcs:config(AccessKeyId, SecretAccessKey, CS_http_port),
+
+    lager:debug("cs_src_root = ~p", [rtdev:relpath(cs_src_root)]),
+    %% NOTE: This 'riak_cs_root' path must appear in
```
Comment (kellymclaughlin) [inline riak_test/tests/external_client_tests.erl]: I think `riak_cs_root` here should be `cs_src_root`.

**Comment by kellymclaughlin** [comment]
The documentation I would suggest is to add a comment about the requirement for `cs_src_root` in the `riak_test` config file to `riak_test/README.md` and also add it to the example config file given there. That was all I had to add to get it working.

**Comment by slfritchie** [comment]
Kelly, I think that your review comments have been addressed now.  Do you have time for another peek?

**Comment by kellymclaughlin** [comment]
We have both solved the problem of wanting to set per-test config options, but slightly differently. If it's ok with you I will rebase this branch and work on sorting out the conflicts and make everything play nice together.

**Comment by slfritchie** [comment]
Kelly, sorry if I overlooked some existing code!  Go ahead.

**Comment by kellymclaughlin** [comment]
I've rebased and adjusted the `external_client_tests` module as needed. The tests are all passing for me now so +1 to merge if you are ok with my changes.

**Comment by slfritchie** [comment]
I liked skipping the extra time required to knit and tear down a 4 node Riak cluster, but ... not a big deal.  Merging.

**Pull request merged**